plugins/mcreator-core/datalists/blocksitems.yaml

```diff
@@ -5954,4 +6446,36 @@
 - Items.HONEY_BOTTLE:
   readable_name: "Honey Bottle"
   type: item
-  texture: HONEY_BOTTLE
\ No newline at end of file
+  texture: HONEY_BOTTLE
+
+- Items.GLOW_BERRIES:
+  readable_name: "Glow Berries"
+  type: item
+
+- Items.GLOW_INK_SAC:
+  readable_name: "Glow Ink Sac"
+  type: item
+
+- Items.AMETHYST_SHARD:
+  readable_name: "Amethyst Shard"
+  type: item
+
+- Items.BUNDLE:
+  readable_name: "Bundle"
+  type: item
+
+- Items.RAW_IRON:
+  readable_name: "Raw Iron"
+  type: item
+
+- Items.RAW_GOLD:
+  readable_name: "Raw GOld"
+  type: item
+
+- Items.RAW_COPPER:
+  readable_name: "Raw Copper"
+  type: item
+
+- Items.SPYGLASS:
+  readable_name: "Spyglass"
+  type: item
```

Comment:
These blocks are probably missing from the list:

- WATER_CAULDRON
- LAVA_CAULDRON
- POWDER_SNOW_CAULDRON
---
- LIGHT
---
- CANDLE_CAKE
- WHITE_CANDLE_CAKE
- ORANGE_CANDLE_CAKE
- MAGENTA_CANDLE_CAKE
- LIGHT_BLUE_CANDLE_CAKE
- YELLOW_CANDLE_CAKE
- LIME_CANDLE_CAKE
- PINK_CANDLE_CAKE
- GRAY_CANDLE_CAKE
- LIGHT_GRAY_CANDLE_CAKE
- CYAN_CANDLE_CAKE
- PURPLE_CANDLE_CAKE
- BLUE_CANDLE_CAKE
- BROWN_CANDLE_CAKE
- GREEN_CANDLE_CAKE
- RED_CANDLE_CAKE
- BLACK_CANDLE_CAKE
---
- CAVE_VINES_PLANT

Reply:
I saw them when I made mappings, but I didn't want to include them because those candles are only used when they are on a cake, so you can't really use them with something else.

Reply:
We should include all blocks and items I think

Reply:
> I saw them when I made mappings, but I didn't want to include them because those candles are only used when they are on a cake, so you can't really use them with something else.

We should still add them so that players can, for example, place them with procedures.

Reply:
![image](https://user-images.githubusercontent.com/38427877/130331852-960fc527-7c59-4966-8252-04ebe7cfface.png)

Reply:
> 
> 
> CAVE_VINES_PLANT: ![image](https://user-images.githubusercontent.com/38427877/130331852-960fc527-7c59-4966-8252-04ebe7cfface.png)

It should still be added, these are different block in the source code.

Reply:
I'm maybe stupid sometimes, but CAVE VINES PLANT and Cave Vines Plant is the same thing.

Reply:
![image](https://user-images.githubusercontent.com/16374228/130365900-6f2c246d-6595-47c8-9b44-90e42f0b8db0.png)

It does not seem like it in the code

Reply:
What's the problem? I don't understand what's your point. The data list entry is CAVE_VINES and in official mappings it's CAVE_VINES_PLANT. What's the problem?

Reply:
I don't know what the conversation was about, I just pointed that out. From the screenshot above, it seems same entry is added twice with different names which is not ok, but not sure if this is what this convo was about ;)

Reply:
Ok my bad, I didn't see the entry before.